docsk8s/setting-up/ls-k8s-persistent-storage.asciidoc

```diff
@@ -1,16 +1,267 @@
 [[ls-k8s-persistent-storage]]
-=== Persistent storage requirements
+=== Stateful {ls}
 
 WARNING: This documentation is still in development and may be changed or removed in a future release.
 
-Some snazzy intro text...
+In some scenarios, {ls} need to persist data to disk: 
 
-[[persistent-storage-pq-dlq]]
-==== PQ and DLQ for {ls} core
+- Enable persistent queue to absorb bursts of events 
+- Enable dead letter queue for the corrupted events that cannot be processed.
+- Some plugins save and resume the state of work
 
-There a few factors to help you decide whether to configure your {ls} core instance to use either a persistent queue {PQ} or a dead letter queue (DLQ).
+Kubernetes scheduler can shutdown pods at anytime and spawn the process to another node. To against data loss during unexpected termination, use `StatefulSet` to deploy {ls}.
```

Comment:
What do you think about giving additional context, and stating that we should use `StatefulSet` instead of `Deployment`

```suggestion
In these cases, we need to ensure that we can preserve state, and take account that the Kubernetes scheduler can shutdown pods at anytime and spawn the process to another node. To enable this, we define our {ls} deployment using `StatefulSet` rather than `Deployment`.
```

Reply:
By additional context, I meant making it clear that `StatefulSet` was a replacement for `Deployment` and indicating the changes that need to be made, and your changes do this